geode-junit/src/main/java/org/apache/geode/internal/AvailablePortHelper.java

```diff
@@ -38,6 +34,23 @@
  * further calls to getRandomAvailablePort.
  */
 public class AvailablePortHelper {
+  static AtomicInteger currentMembershipPort;
```

Comment:
I recommend making these private final. You can still change the values of them in the static block.